server/db/models.py

```diff
@@ -158,25 +179,72 @@
 
 moderation_report = Table(
    'moderation_report', metadata,
-   Column('id', Integer, primary_key=True),
-   Column('reporter_id', ForeignKey('login.id'), nullable=False),
-   Column('report_description', Text),
-   Column('game_id', ForeignKey('game_stats.id'), index=True),
-   Column('report_status', Enum('AWAITING', 'PROCESSING', 'COMPLETED', 'DISCARDED'), nullable=False),
+   Column('id',                     Integer,                        primary_key=True),
+   Column('reporter_id',            ForeignKey('login.id'),         nullable=False),
+   Column('report_description',     Text),
+   Column('game_id',                ForeignKey('game_stats.id'),    index=True),
    Column('game_incident_timecode', String(100)),
-   Column('moderator_notice', Text),
+   Column('moderator_notice',       Text),
    Column('moderator_private_note', Text),
-   Column('last_moderator', ForeignKey('login.id'), index=True),
-   Column('create_time',   TIMESTAMP,  nullable=False),
-   Column('update_time',   TIMESTAMP,  nullable=False)
+   Column('last_moderator',         ForeignKey('login.id'),         index=True),
+   Column('create_time',            TIMESTAMP,                      nullable=False),
+   Column('update_time',            TIMESTAMP,                      nullable=False),
+   Column(
+       'report_status',
+       Enum('AWAITING', 'PROCESSING', 'COMPLETED', 'DISCARDED'),
+       nullable=False
+    )
```

Comment:
You know, you didn't made it better

Reply:
Yea, in this file I'd let lines go as long as they need so we have 1 table column per line